servlet-dist-legacy/pom.xml

```diff
@@ -175,6 +175,91 @@
                             </execution>
                         </executions>
                     </plugin>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-antrun-plugin</artifactId>
+                        <version>1.7</version>
+                        <executions>
+                            <execution>
+                                <phase>prepare-package</phase>
+                                <configuration>
+                                    <tasks>
+                                        <filelist id="licenses.xml.list" dir="${basedir}/target/${server.output.dir.prefix}-servlet-${server.output.dir.version}/docs/licenses"
+                                                  files="servlet-feature-pack-licenses.xml,core-feature-pack-licenses.xml"/>
+
+                                        <pathconvert pathsep="&#xA;" property="license-files-list-items" refid="licenses.xml.list">
+                                            <chainedmapper>
+                                                <mapper type="flatten"/>
+                                                <globmapper from="*" to="&lt;item&gt;*"/>
+                                                <globmapper from="*" to="*&lt;/item&gt;"/>
+                                            </chainedmapper>
+                                        </pathconvert>
+                                        <echo file="${basedir}/target/${server.output.dir.prefix}-servlet-${server.output.dir.version}/docs/licenses/licenses.xml">&lt;?xml version="1.0" encoding="UTF-8"?&gt;&#xA;&lt;list&gt;&#xA;</echo>
+                                        <echo file="${basedir}/target/${server.output.dir.prefix}-servlet-${server.output.dir.version}/docs/licenses/licenses.xml" append="true">${license-files-list-items}&#xA;</echo>
+                                        <echo file="${basedir}/target/${server.output.dir.prefix}-servlet-${server.output.dir.version}/docs/licenses/licenses.xml" append="true">&lt;/list&gt;</echo>
+                                    </tasks>
+                                </configuration>
+                                <goals>
+                                    <goal>run</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                    </plugin>
+                    <plugin>
+                        <groupId>org.codehaus.mojo</groupId>
+                        <artifactId>xml-maven-plugin</artifactId>
+                        <version>1.0.1</version>
```

Comment:
The version should be a property, if it's not already, in the main pom.

Reply:
Fixed.